src/ray/core_worker/core_worker.cc

```diff
@@ -686,6 +689,14 @@ void CoreWorker::Shutdown() {
   }
 }
 
+void CoreWorker::NotifyRaylet() {
```

Comment:
nit: can we rename this to ConnectToRaylet?

Reply:
Technically, this function doesn't "connect" to raylet, and that's why I chose this name (connection happens when the client class is created, which happens in the constructor). But I think it is still okay to use this name. I will follow your suggestion, but if you have a better name in mind, plz let me know!